demo-apps/cf-android/app/src/main/java/org/eclipse/californium/examples/MainActivity.java

```diff
@@ -89,6 +107,15 @@ protected void onPreExecute() {
 
         protected CoapResponse doInBackground(String... args) {
             CoapClient client = new CoapClient(args[0]);
+            if (isSecure) {
```

Comment:
Means: only if the localhost dtls service is running, dtls could be used. Do yo see something else, which may determine, if you want to use dtls/coaps or plain/coap?